inbm/dispatcher-agent/tests/unit/sota/test_snapshot.py

```diff
@@ -113,7 +113,7 @@ def test_rollback_and_delete_snap_skipped_succeeds(self) -> None:
         dispatcher_broker = Mock()
         trtl = Mock()
 
-        ubuntu_snapshot = DebianBasedSnapshot(trtl, "command", dispatcher_broker, "", True)
+        ubuntu_snapshot = DebianBasedSnapshot(trtl, "command", dispatcher_broker, "", True, "Y")
         ubuntu_snapshot._rollback_and_delete_snap()
 
         assert dispatcher_broker.telemetry.call_count > 0
```

Comment:
**Suggestion:** Use assertTrue to check if a string is in the message for consistency with unittest practices. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self.assertIn("skipped", message)
```